bridge-deploy/bridge_deploy/foreign.py

```diff
@@ -0,0 +1,35 @@
+from typing import Dict, NamedTuple
+from web3.contract import Contract
+from deploy_tools.deploy import (
+    deploy_compiled_contract,
+    increase_transaction_options_nonce,
+)
+
+from bridge_deploy.utils import load_build_contract
+
+
+class DeployedForeignBridgeResult(NamedTuple):
+    foreign_bridge: Contract
+    foreign_bridge_block_number: int
+
+
+def deploy_foreign_bridge_contract(
+    *, web3, transaction_options: Dict = None, private_key=None
+):
+
+    if transaction_options is None:
+        transaction_options = {}
+
+    latest_block = web3.eth.getBlock("latest")
+
+    foreign_bridge_src = load_build_contract("ForeignBridge")
+    foreign_bridge_contract = deploy_compiled_contract(
+        abi=foreign_bridge_src["abi"],
+        bytecode=foreign_bridge_src["bytecode"],
+        web3=web3,
+        transaction_options=transaction_options,
+        private_key=private_key,
+    )
+    increase_transaction_options_nonce(transaction_options)
+
+    return DeployedForeignBridgeResult(foreign_bridge_contract, latest_block.number)
```

Comment:
I still don't get why we do this weird thing where we give `web3` to a function and it returns `web3.eth.getBlock("latest")` to us in a weird `DeployedForeignBridgeResult` output. Just return the contract in this function and get the latest block number in the cli.

Reply:
Otherwise the rest is fine

Reply:
I just took results of the home deployment as template and don't start to do it in another way. I though you would have discussed this at the responsive PR.
So if I do so, should I also adjust the home bridge deployment part regarding this part? @saschagoebel your comment here please as well.

Reply:
I went back to look for his comment on the other PR that I did not see before: 
"We don't need the block number for the initialize, but for the docker-compose based validator application."

Reply:
Fetching the latest block could indeed be in the cli function instead of the deploy function, as it's only printed, not actually used.
Sorry, my mistake.